doc/Extensions/Oxidized.md

```diff
@@ -57,13 +57,13 @@ LibreNMS is able to reload the Oxidized list of nodes, each time a device is add
 To do so, edit the option in Global Settings>External Settings>Oxidized Integration or add the following to your config.php.
 
 ```php
-$config['oxidized']['reload_nodes'] = TRUE;
+$config['oxidized']['reload_nodes'] = true;
```

Comment:
Absolutely, I've not found a good way to still have the config options in the doc but 'hide' them.